app/elements/event-sponsors.mjs

```diff
@@ -0,0 +1,33 @@
+export default function eventSponsors({ html, state = {} }) {
+    const { store = {} } = state
+    const { eventSponsors } = store
+
+      return html`
+        <style>
+          :host div {
+            padding: 16px;
+          }
+
+          :host img {
+            height: 80px;
+          }
+          @media only screen and (min-width: 768px) {
+            :host {
+              display: flex;
```

Comment:
The list of events looks a little funky on a desktop browser. It's not aligned with the title. Probably not a blocking issue, but is there an easy fix?
![image](https://github.com/seattlejs/seattlejs.com/assets/40310772/dc3f233d-ba51-4ae4-ab43-0d98a48de815)

Reply:
whoops, Carter already mentioned this here: https://github.com/seattlejs/seattlejs.com/pull/118#issuecomment-1739834611